src/main/g8/src/main/scala/$package__packaged$/ServerStream.scala

```diff
@@ -0,0 +1,18 @@
+package $package$
+
+import cats.effect.{ConcurrentEffect, Effect, ExitCode, IO, IOApp, Timer, ContextShift}
+import cats.implicits._
+import org.http4s.server.blaze.BlazeServerBuilder
+import org.http4s.implicits._
+import fs2.Stream
+
+object ServerStream {
+  def helloWorldApp[F[_]: Effect] = HelloWorldRoutes.routes[F].orNotFound
+
+  def stream[F[_]: ConcurrentEffect](implicit T: Timer[F], C: ContextShift[F]): Stream[F, Nothing] =
```

Comment:
What if this took an `HttpApp[F]` as a parameter?  Why does this depend on `HelloWorldRoutes` at all?

Reply:
Generally this would be where you build the `HttpApp`. It's just not necessary in this case. Would you like me to extend this such that it does things that need that. Perhaps with a client?

Reply:
My question is whether this object should be coupled to HelloWorldRoutes.  Is the argument that this is the assembly that wires together potentially more resources?  I might buy that, or else this is merely another set of defaults for BlazeServerBuilder.

Maybe I would like this better as HelloWorld, and the IOApp as HelloWorldApp.  Maybe my problem is how (specific) HelloWorldServer depends on (generic) ServerStream depends on (specific) HelloWorldRoutes.  Maybe I like the structure and not the names.

I confess I'm thinking out loud.